### PR TITLE
AUC performance comparison between two methods using multiple iterati…

### PR DESCRIPTION
**Pull request by ZhenjiangFan**
…ons of true labels and corresponding predicted labels.

## PR checklist

- [ *] "closes #0000" is in the body of the PR description to [link the related issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
- [ *] new and changed code is [tested](https://matplotlib.org/devdocs/devel/testing.html)
- [ ] *Plotting related* features are demonstrated in an [example](https://matplotlib.org/devdocs/devel/document.html#write-examples-and-tutorials)
- [* ] *New Features* and *API Changes* are noted with a [directive and release note](https://matplotlib.org/devdocs/devel/api_changes.html#announce-changes-deprecations-and-new-features)
- [ *] Documentation complies with [general](https://matplotlib.org/devdocs/devel/document.html#write-rest-pages) and [docstring](https://matplotlib.org/devdocs/devel/document.html#write-docstrings) guidelines

### CODE REVIEW

**Comment by timhoffm** [comment]
Thanks for your willingness to contribute. However, I don't think this is a suitable example for matplotlib because

- sklearn is not a doc dependency, so you cannot use it in the example
- most of the example is on data creation and processing. The plotting itself is quite simple: some `plot` and `fill_between` and a bit of common configuration. So overall this does not teach much about plotting with matplotlib.

I haven't looked into the details of your plot, but you could ask sklearn, whether it fits [their examples](https://scikit-learn.org/stable/auto_examples/index.html).

**Comment by tacaswell** [comment]
I agree with @timhoffm .  We are not going to pick up sklearn as a docs dependency and on a quick read this appears that most of the code is related to the ML side of things not the plotting. 

Thank you for your contribution @ZhenjiangFan , but I am going to close this without merging.

**Comment by ZhenjiangFan** [comment]
Thank you, Tim and Tom, for your time to review my request. I appreciate it.

Have a great rest of your day.

Best,

Fan
